bin/postgres-ha/bootstrap/pre-bootstrap.sh

```diff
@@ -414,10 +414,7 @@ build_bootstrap_config_file() {
         if [[ "${bootstrap_cluster_source}" != "${bootstrap_cluster_target}" ]];
         then
             echo_info "Disabling archive mode for bootstrap method ${PGHA_BOOTSTRAP_METHOD}"
```

Comment:
Let's update the message:

```suggestion
            echo_info "Disabling archiving for bootstrap method ${PGHA_BOOTSTRAP_METHOD}"
```

Reply:
Thanks, good idea.  Message updated!